recipe/meta.yaml

```diff
@@ -1,40 +1,36 @@
-{% set base_version = "3.7.3" %}
-{% set version = "3.7.3" %}
+{% set base_version = "3.8.0" %}
+{% set version = "3.8.0b3" %}
 {% set linkage_nature = os.environ.get('PY_INTERP_LINKAGE_NATURE', '') %}
 {% if linkage_nature != '' %}
   {% set linkage_nature = "_" ~ linkage_nature %}
 {% endif %}
 
 package:
   name: python
-  version: {{ version }}
+  version: {{ base_version }}
```

Comment:
I am opposed to faking the version here.

Reply:
OK, please advise on better suggestions for betas with version comparison considerations, err, considered (carefullly!).